packages/deployment/upgrade-test/Makefile

```
@@ -13,11 +13,8 @@ BUILD = docker build --progress=plain $(BUILD_OPTS) \
 	--build-arg BOOTSTRAP_MODE=$(BOOTSTRAP_MODE) --build-arg DEST_IMAGE=$(DEST_IMAGE) \
 	-f Dockerfile upgrade-test-scripts
 
-propose-agoric-upgrade-12:
-	$(BUILD) --target propose-agoric-upgrade-12 -t $(REPOSITORY):propose-agoric-upgrade-12$(TAG_SUFFIX)
-
-agoric-upgrade-12: propose-agoric-upgrade-12
-	$(BUILD) --target agoric-upgrade-12 -t $(REPOSITORY):agoric-upgrade-12$(TAG_SUFFIX)
+wallet-factory-upgrade:
+	$(BUILD) --target wallet-factory-upgrade -t $(REPOSITORY):wallet-factory-upgrade
```

being last, it's the default target
```suggestion
	$(BUILD) -t $(REPOSITORY):wallet-factory-upgrade
```